CSharp/DevVmPowershell/Helpers/Implementations/RelativityVersionHelper.cs

```diff
@@ -11,29 +12,31 @@ namespace Helpers.Implementations
 {
 	public class RelativityVersionHelper : IRelativityVersionHelper
 	{
-		private ServiceFactory ServiceFactory { get; }
+		private string InstanceAddress { get; }
+		private string AdminUsername { get; }
+		private string AdminPassword { get; }
+		private IRestHelper RestHelper { get; set; }
 
-		public RelativityVersionHelper(IConnectionHelper connectionHelper)
+		public RelativityVersionHelper(IRestHelper restHelper, string instanceAddress, string adminUsername, string adminPassword)
 		{
-			ServiceFactory = connectionHelper.GetServiceFactory();
+			InstanceAddress = instanceAddress;
+			AdminUsername = adminUsername;
+			AdminPassword = adminPassword;
+			RestHelper = restHelper;
 		}
 
-		public void ConfirmInstallerAndInstanceRelativityVersionAreEqual(string installerRelativityVersion)
+		public async Task ConfirmInstallerAndInstanceRelativityVersionAreEqual(string installerRelativityVersion)
 		{
-			try
+			HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+			HttpResponseMessage response = await RestHelper.MakePostAsync(httpClient, Constants.Connection.RestUrlEndpoints.InstanceDetailsService.EndpointUrl, "");
```

Comment:
Doesn't this should be a GET call as we are retrieving data?

- [ ] Use GET

Reply:
You use POST for this endpoint, as listed in this page: https://platform.relativity.com/9.5/Content/REST_API/Resource_Tasks/Retrieve_the_version_of_a_Relativity_instance.htm